source/class/qx/ui/core/MPlacement.js

```diff
@@ -376,26 +377,32 @@ qx.Mixin.define("qx.ui.core.MPlacement", {
         );
       }
 
-      var coords =
+      const coords =
```

Comment:
please do not use `const` in this instance - firstly, changing `let` to `const` is not relevant to the change that this PR is about, and secondly the value referred to is not "constant" - it is a variable.

Even if it was a constant, the coding standards would expect the variable name to show that eg `const SECONDS_PER_HOUR = 60 * 60`

Reply:
I've changed it back to `var`.